src/Component/TopBtn.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import "./TopBtn.scss"
+
+export default class TopBtn extends Component {
+    render() {
+        return (
+            <>
+                <button type="button" class="btn-to-top-arrow">
+                </button>
```

Comment:
이런식으로 button 태그로 enclosed 된 내용이 따로 없다면 JSX 에서는 아래처럼 self-closing 태그로 변경하실 수 있습니다.
```suggestion
                <button type="button" class="btn-to-top-arrow" />
```